### DISCUSSION TITLE
Is there any computercraft support?

### DISCUSSION THREAD

**Discussion by AceOfZpades** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1464] [Q&A] [answered]
Im trying to make a monitor that tracks a Zirnox's temperature and pressure but I cant find much about compatibility between ComputerCraft and HBM, only a wiki page about OpenComputers

**Comment by gmhs022** [comment, marked as answer]
No, as far as I know the methods only work with OpenComputers.
